test/extended/router/wildcard.go

```diff
@@ -44,17 +48,26 @@ var _ = g.Describe("[Conformance][Area:Networking][Feature:Router]", func() {
 	})
 
 	g.Describe("The default ClusterIngress", func() {
-		g.It("should support default wildcard reencrypt routes through external DNS", func() {
-			execPodName := exutil.CreateExecPodOrFail(oc.AdminKubeClient().Core(), ns, "execpod")
-			defer func() { oc.AdminKubeClient().Core().Pods(ns).Delete(execPodName, metav1.NewDeleteOptions(1)) }()
-
-			g.By(fmt.Sprintf("deploying a service using a reencrypt route using only defaults"))
-			err := oc.Run("create").Args("-f", fixture).Execute()
+		g.It("should enable a route whose default host can be resolved by external DNS", func() {
+			g.By("creating a simple route with a default hostname")
+			route := &routev1.Route{
+				ObjectMeta: metav1.ObjectMeta{
+					Namespace: ns,
+					Name:      "simple",
+				},
+				Spec: routev1.RouteSpec{
+					To: routev1.RouteTargetReference{
+						Name: "nowhere",
+					},
+				},
+			}
+			_, err := oc.RouteClient().Route().Routes(ns).Create(route)
 			o.Expect(err).NotTo(o.HaveOccurred())
```

Comment:
Should we do a `defer func() { oc.RouteClient().Route().Routes(ns).Delete(route.Name, metav1.NewDeleteOptions(0)) }()`?

Reply:
The whole test namespace is deleted so I'm not sure why it matters

Reply:
We delete the pod explicitly, but fair enough.